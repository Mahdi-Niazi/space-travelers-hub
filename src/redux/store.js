import { configureStore } from '@reduxjs/toolkit';

import missionReducer from './slices/MissionsSlice';

const store = configureStore({
  reducer: {
    missions: missionReducer,
  },
});

export default store;
