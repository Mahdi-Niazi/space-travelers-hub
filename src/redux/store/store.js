import { configureStore } from '@reduxjs/toolkit';
import rocketsReducer from '../slices/RocketsSlice';

const store = configureStore({
  reducer: {
    rockets: rocketsReducer,
  },
});

export default store;
