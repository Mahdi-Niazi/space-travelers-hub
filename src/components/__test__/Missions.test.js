import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import Mission from '../Missions';
import store from '../../redux/store';

describe('Mission component', () => {
  it('should match snapshot', () => {
    const tree = renderer.create(
      <Provider store={store}>
        <Mission />
      </Provider>,
    );
    expect(tree).toMatchSnapshot();
  });
});
