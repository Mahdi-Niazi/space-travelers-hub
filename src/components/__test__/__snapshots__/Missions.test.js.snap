// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Mission component should match snapshot 1`] = `
<div
  className="table-container"
>
  <table
    className="mission-table"
  >
    <thead>
      <tr>
        <th>
          Missions
        </th>
        <th>
          Descriptions
        </th>
        <th
          className="status"
        >
          Status
        </th>
        <th
          className="status"
        >
          Action
        </th>
      </tr>
    </thead>
    <tbody />
  </table>
</div>
`;
